# This patch is Solaris specific and thus has not been contributed upstream.

--- /dev/null						2013-06-18 15:31:00.000000000 -0700
+++ sendmail-8.14.9/devtools/Site/site.SunOS.5.12.m4	2013-03-26 07:18:05.675345928 -0700
@@ -0,0 +1,7 @@
+APPENDDEF(`conf_sendmail_ENVDEF', `-DTCPWRAPPERS -DSTARTTLS -D_FFR_LOCAL_DAEMON -DSUN_EXTENSIONS -DVENDOR_DEFAULT=VENDOR_SUN')
+APPENDDEF(`conf_sendmail_LIBS', `-lwrap -lssl -lcrypto')
+APPENDDEF(`confMAPDEF', `-DLDAPMAP')
+define(`confDEPEND_TYPE', `Solaris')
+define(`confLIBS', `-lldap -lc')
+define(`confMKDIR', `/usr/bin/mkdir')
+define(`sendmailTARGET_LINKS', `')
